.changeset/clean-mirrors-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+**feat**: SwapSettings by @0xAlec #1051
```

Comment:
Let's eplain a bit more what this component will help add to the developer experience.

Reply:
Updated